api/mm/make.build

```diff
@@ -0,0 +1,59 @@
+#------------------------------------------------------------------------------#
+#  DFTB+: general package for performing fast atomistic simulations            #
+#  Copyright (C) 2017  DFTB+ developers group                                  #
+#                                                                              #
+#  See the LICENSE file for terms of usage and distribution.                   #
+#------------------------------------------------------------------------------#
+
+TARGETS = libdftb+.a
+
+.PHONY: all
+all: $(TARGETS)
+
+# Include common makefile
+SRCDIRS := $(ROOT)/api/mm
+SRCDIRS += $(wildcard $(ROOT)/prog/dftb+/lib_*)
+SRCDIRS += $(ROOT)/prog/dftb+/include
+INCDIRS := $(ROOT)/prog/dftb+/include
+include $(ROOT)/prog/dftb+/make.common
+
+libdftb+.a: $(dftbplus.o)
+	ar r $@ $(filter-out _%,$^)
```

Comment:
archive tool choice (along with its options) should probably be set by the sys make files

Reply:
I agree. However, currently all external components have hardcoded "ar" command lines. Again, a later change to CMake should resolve this issue, as CMake would set the correct command line automatically.

Reply:
OK defer to cmake